v2.0/experimental-audit.md

```diff
@@ -89,12 +89,14 @@ If your deployment requires particular lifecycle and access policies for audit l
 
 Let's say you have a  `customers` table that contains personally identifiable information (PII). To turn on audit logs for that table, run the following command:
 
+{% include copy-clipboard.html %}
 ~~~ sql
 ALTER TABLE customers EXPERIMENTAL_AUDIT SET READ WRITE;
 ~~~
 
 Now, every access of customer data is added to the audit log with a line that looks like the following:
 
+{% include copy-clipboard.html %}
```

Comment:
Can you please remove the copy-to-clipboard button from this block?  The button is only for executable code per our [style guide](https://github.com/cockroachdb/docs/blob/master/STYLE.md#code) (see the section **Copy to Clipboard Button**).  Since this code block is from a log file, it doesn't need the button.